libs/ui/src/components/NeoButton/NeoButton.scss

```diff
@@ -4,10 +4,14 @@
 
 .o-btn {
   &.is-neo {
-    @apply rounded-none font-inherit py-2;
+    @apply  h-10 min-w-10 rounded-none font-inherit py-2;
```

Comment:
```suggestion
    @apply  h-10 rounded-none font-inherit py-2;
```

not the right fix for 

> gallery item in share <img alt="image" width="167" src="https://private-user-images.githubusercontent.com/90852205/323834696-155348e8-e2b5-4f5f-91fd-f405761d8611.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQzNzM4MTIsIm5iZiI6MTcxNDM3MzUxMiwicGF0aCI6Ii85MDg1MjIwNS8zMjM4MzQ2OTYtMTU1MzQ4ZTgtZTJiNS00ZjVmLTkxZmQtZjQwNTc2MWQ4NjExLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI5VDA2NTE1MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYwOTY3YzcxZjAzOGNlMTAwMWQxZGIxNDVhOTE0MjgzYWMyYjc2YTQwNTMwNTMyOGNkN2YyZDdlYmQ0NDM2MjYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0._iCdQ2GfSmaw-j4DGPlnuEWeXewRaIpOHm51jzIqIoU">
> 
> * perfect square please
> * 40x40


this is the issue

![CleanShot 2024-04-29 at 11 59 37@2x](https://github.com/kodadot/nft-gallery/assets/44554284/3a024b0f-69af-4bb9-9c15-bae4c3c90874)

Reply:
I think we should remove the icon action class instead since min-w-10 and h-10 is the way to go now according to figma? minimum is w-10

Reply:
but yes lets to all the followup fixes in another issue

Reply:
that class is used somewhere else , check to see what is the appropriate fix 

![CleanShot 2024-04-30 at 10 20 41@2x](https://github.com/kodadot/nft-gallery/assets/44554284/1fdd939b-9d82-4fe5-a749-71b88c116a2e)